services/src/main/java/org/keycloak/services/clientpolicy/condition/ClientUpdaterSourceHostsCondition.java

```diff
@@ -85,7 +85,11 @@ public ClientPolicyVote applyPolicy(ClientPolicyContext context) throws ClientPo
     private boolean isHostMatched() {
         String hostAddress = session.getContext().getConnection().getRemoteAddr();
```

Comment:
Should we use `getRemoteHost` here instead? Or if we're really validating the IP, why are we using `getRemoteHost` there?

Reply:
For display / logging purposes we're using the host.